v1/ao/agent_test.go

```diff
@@ -33,3 +37,18 @@ func TestShutdown(t *testing.T) {
 	defer cancel()
 	assert.False(t, WaitForReady(ctx))
 }
+
+func TestSetLogOutput(t *testing.T) {
+	oldLevel := GetLogLevel()
+	_ = SetLogLevel("DEBUG")
```

Comment:
Thanks for pointing it out. It was added because otherwise there is an `Unhandled error` warning generated by the Goland code inspection. The warning may be too strong.